plugins/catalog-backend-module-msgraph/api-report.md

```diff
@@ -111,10 +111,7 @@ export class MicrosoftGraphClient {
     query?: ODataQuery,
     queryMode?: 'basic' | 'advanced',
   ): AsyncIterable<T>;
-  requestRaw(
-    url: string,
```

Comment:
Wonder if it makes sense to just keep this as a `string` here instead, and do `new URL(...)` in the implementation, otherwise this is a breaking change.

Reply:
True, I will change this